modules/standard/AutoMath.chpl

```diff
@@ -1111,7 +1023,7 @@ module AutoMath {
      fewer conditionals will be evaluated at run time.
   */
   pragma "last resort"
-  @deprecated(notes="In an upcoming release 'divceil' will no longer be included by default.  Please 'use' or 'import' the :mod:`Math` module to prepare for this move, noting that its name has changed to :proc:`~Math.divCeil` and its argument names have changed to 'x' and 'y'")
+  @deprecated(notes="In an upcoming release 'divceil' will no longer be included by default.  Please 'use' or 'import' the :mod:`Math` module to prepare for this move, noting that its name has changed to `divCeil`` and its argument names have changed to 'x' and 'y'")
```

Comment:
You should remove the extra ` at the end of the divCeil reference or add the missing one at the front.  I'm surprised it didn't cause problems.

Reply:
Thinking about it more, I would just hide all the deprecated "by-default" versions from the documentation.  I think maintaining them made a lot more sense when they were documented in separate pages, but having them in a single page where the version that actually lives in Math is the one we want to keep just makes things confusing.

Thankfully, I think most of them could be removed entirely before 1.33 or whatever the next version ends up being, but iirc at least the divceil ones have to continue to warn for another release.